scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -1,23 +1,23 @@
 (ns cook.kubernetes.compute-cluster
-  (:require [clj-time.core :as t]
-            [clj-time.periodic :as tp]
-            [clojure.core.async :as async]
+  (:require [clojure.core.async :as async]
             [clojure.set :as set]
             [clojure.tools.logging :as log]
             [cook.compute-cluster :as cc]
             [cook.config :as config]
             [cook.kubernetes.api :as api]
             [cook.kubernetes.controller :as controller]
             [cook.pool]
-            [cook.mesos.sandbox :as sandbox]
-            [cook.scheduler.scheduler :as scheduler]
             [datomic.api :as d]
             [plumbing.core :as pc])
-  (:import (io.kubernetes.client ApiClient)
+  (:import (com.google.auth.oauth2 GoogleCredentials)
+           (io.kubernetes.client ApiClient)
            (io.kubernetes.client.apis CoreV1Api)
-           (io.kubernetes.client.models V1PodStatus V1ContainerState)
            (io.kubernetes.client.util Config)
-           (java.util UUID)))
+           (java.io FileInputStream File)
+           (java.util UUID)
+           (java.util.concurrent Executors ScheduledExecutorService TimeUnit)
+           (com.squareup.okhttp.logging HttpLoggingInterceptor HttpLoggingInterceptor$Level)
```

Comment:
lexicographical order.

Reply:
Actually a lot of these were unused from debugging, removed.